hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/utils/HoodieInputFormatUtils.java

```diff
@@ -438,14 +437,26 @@ public static HoodieMetadataConfig buildMetadataConfig(Configuration conf) {
         if (LOG.isDebugEnabled()) {
           LOG.debug("Hoodie Metadata initialized with completed commit instant as :" + metaClient);
         }
-
         HoodieTimeline timeline = HoodieHiveUtils.getTableTimeline(metaClient.getTableConfig().getTableName(), job, metaClient);
```

Comment:
can we restrict the timeline based on max commit time? pass the boundedtimeline to filesystem view creation would fit better with existing flow i think

Reply:
Ack